src/config.js

```diff
@@ -0,0 +1,3 @@
+export const ORGANIZATION = process.env.REACT_APP_ORGANIZATION || 'org-name';
```

Comment:
I think the `config.js` should stay as it is, and the code that gets value from that file _or_ env variables should lie in `github.js`.

E.g.

```js
function getConfigValues() {
  let ORGANIZATION = process.env.ORGANIZATION;
  // ...
  if (!ORGANIZATION) {
    const config = require('./config');
    ORGANIZATION = config.ORGANIZATION;
    // ...
  }
  return { ORGANIZATION, ... }; 
}
```

Reply:
Fair enough, I'll make this change in the near future.

Reply:
Got it working with a trick - will send you a PR

Reply:
So the trick was to declare "./config" as a string but that breaks things when you try to use a `config.js`:
![image](https://user-images.githubusercontent.com/32899150/47897989-fbd99900-dec7-11e8-8b3c-578625aec307.png)

Not sure how to proceed. Alternatively do no changes and instead, write out the `config.js` when starting as a Docker container.